src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private String color = colorSupplier.getRandomColor();
+
+    public Ball getRandomBall() {
+        int number = new Random().nextInt(100);
+        return new Ball(color, number);
+    }
```

Comment:
```suggestion
    private ColorSupplier colorSupplier = new ColorSupplier();
    private Random random = new Random();

    public Ball getRandomBall() {
        int number = random.nextInt(100);
        String color = colorSupplier.getRandomColor();
        return new Ball(color, number);
    }
```

Тут тоже самое. Таким образом нам не нужно будет создавать новый экзмепляр `Random` и `ColorSupplier` (через слово `new`) каждый раз, когда мы вызываем метод `getRandomBall()`

Reply:
Спасибо за ревью)